subsystem/swerveDriveTrain.py

```diff
@@ -133,12 +134,29 @@ def drive(self, xSpeed: float, ySpeed: float, rot: float, fieldRelative: bool):
         for mod, state in zip(self.swerveModules, swerveModuleStates):
             mod.setSwerveModuleState(state, self.kMaxVelocityMPS)
 
+        self.updateOdometry()
```

Comment:
Should this be in this changeset or is updateOdometry() from another branch? If the latter, we can help get this cleaned up so this is ONLY for telemetry.